documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -1086,6 +1095,13 @@ dcc.clustering().partitionHandling()
                     .whenSplit(PartitionHandling.ALLOW_READ_WRITES)
                     .mergePolicy(new CustomMergePolicy());
 ----
+==== Deploy custom merge policies to a Infinispan server instance
+To utilise a custom EntryMergePolicy implementation on the server, it's necessary for the implementation class(es) to be deployed to the server. This is accomplished by utilising the java service-provider convention and packaging the class files in a jar which has a META-INF/services/org.infinispan.conflict.EntryMergePolicy file containing the fully qualified class name of the EntryMergePolicy implementation.
+[source]
+----
+# list all necessary implementations of EntryMergePolicy with the full qualified name
+org.example.CustomEntryMergePolicy
+----
```

Comment:
We should state that currently it's only possible for this to work with Compatibility mode enabled.